web/pdf_thumbnail_view.js

```diff
@@ -405,11 +423,13 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
     },
   };
 
+  PDFThumbnailView.cleanup = function() {
```

Comment:
No worries; I suppose that I can see why this looks a bit strange on first sight!

With ES6 classes, this particular code becomes slightly less verbose, since static methods are placed among "regular" ones; using the `static cleanup() { ... }` notation.